drivers/hmis_external_apis/app/views/layouts/hmis_external_apis/external_forms.haml

```diff
@@ -35,10 +35,6 @@
         max-width: 540px;
         margin-bottom: 4rem;
       }
-      .dependent-form-group > .form-group {
-        padding-left : 1.25rem;
-        border-left: 5px solid #eee;
-      }
```

Comment:
This was indentation with a gray vertical bar for conditional items. This is not practical for PIT because there are items that are conditional on value from a question earlier in the form, that should not be indented